task-5/results/.env

```
@@ -0,0 +1,5 @@
+POSTGRESDB_USER=postgres
+POSTGRESDB_ROOT_PASSWORD=123456
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRESDB_ROOT_PASSWORD key should go before the POSTGRESDB_USER key

(UnorderedKey)